kolibri/core/logger/tasks.py

```diff
@@ -25,6 +29,55 @@ def get_filepath(log_type, facility_id, start_date, end_date):
     return filepath
 
 
+def get_valid_logs_csv_filenames():
+    """
+    Returns a set of valid filenames that should exist
+    based on the objects stored in GenerateCSVLogRequest.
+    Any other files except these filenames should be removed.
+    """
+    valid_filenames_set = set()
+    log_requests = GenerateCSVLogRequest.objects.all()
+    for log_request in log_requests:
+        full_path = get_filepath(
+            log_request.log_type,
+            log_request.facility_id,
+            log_request.selected_start_date.strftime("%Y-%m-%d"),
+            log_request.selected_end_date.strftime("%Y-%m-%d"),
+        )
+        valid_filenames_set.add(os.path.basename(full_path))
+    return valid_filenames_set
+
+
+def get_valid_users_csv_filenames():
+    """
+    Returns a set of valid filenames that should exist
+    based on the objects stored in Facility.
+    """
+    valid_filenames_set = set()
+    facilities = Facility.objects.values("id", "name").all()
+    for facility in facilities:
+        file_name = USER_CSV_EXPORT_FILENAMES["user"].format(
+            facility["name"], facility["id"][:4]
+        )
+        valid_filenames_set.add(file_name)
+    return valid_filenames_set
+
+
+def get_valid_filenames():
+    """
+    Returns a union set of valid filenames
+    for log exports and users csv exports.
+    These filenames are valid and will not be
+    cleaned from log_exports_cleanup.
+    """
+    valid_filenames = set()
+    valid_logs_filenames = get_valid_logs_csv_filenames()
+    valid_users_filenames = get_valid_users_csv_filenames()
+    valid_filenames = valid_filenames.union(valid_logs_filenames)
+    valid_filenames = valid_filenames.union(valid_users_filenames)
+    return valid_filenames
```

Comment:
This is nothing major, but I think `valid_filenames` could be set with one line using: `valid_logs_filenames.union(valid_users_filenames)`